lighthouse-core/audits/metrics/first-cpu-idle.js

```diff
@@ -7,6 +7,7 @@
 
 const Audit = require('../audit');
 const i18n = require('../../lib/i18n/i18n.js');
+const ComputedFci = require('../../gather/computed/metrics/first-cpu-idle.js');
```

Comment:
> ComputedFci 🤢

I couldn't think of another name since `SpeedIndex` is already taken by the audit class itself and `ComputedFirstCPUIdle` seemed way too long. I spent exactly 0 time coming up with the name, though, so I'm happy to change :)

Reply:
> ComputedFci 🤢

Unless you mean the capitalization scheme, in which case I WILL DIE ON THIS HILL

Reply:
> Unless you mean the capitalization scheme

Hahaha, that is exactly what I mean :)

Reply:
Then shouldn't below be `FirstCpuIdle`? :)

Reply:
ha, @paulirish caught the same thing :) My defense is that I was merely moving the existing line in the file, `git diff` just can't handle visualizing it :P

Reply:
(also that--even though it totally is the way better way--we gave up on consistency on this particular issue a long time ago because no one manages to remember to enforce their favorite way every time. See also: `XMLHttpRequest`)